### DISCUSSION TITLE
Deleted  .md file from content but I still see it in my site

### DISCUSSION THREAD

**Discussion by shenyuygoodness** [thuliteio/doks#1099] [Q&A]
I created the file directly in the content directory and when I delete it, it still shows alongside its content on my site even after re-running "hugo --gc" several times to clear the hugo cache I still get the same result.

**Comment by pranavdhawale** [comment]
@h-enk can you please help with this also

**Reply by h-enk** [reply]
Clean the Hugo build directories — something like this:

```bash
npx shx rm -rf public resources .hugo_build.lock
```

**Reply by h-enk** [reply]
https://github.com/gethyas/doks/discussions/1150#discussioncomment-9520354

**Reply by pranavdhawale** [reply]
Yep figured it out as well. Thanks alot though.

[#1150 (help me with this also 😞) ](https://github.com/gethyas/doks/discussions/1207#discussioncomment-9520193)
